test/libsolidity/semanticTests/structs/calldata/calldata_struct_to_storage.sol

```diff
@@ -0,0 +1,21 @@
+pragma experimental ABIEncoderV2;
+
+contract C {
+    struct S {
+        uint256 a;
+        uint256 b;
+        bytes2 c;
+    }
+
+    S s;
+
+    function f(S calldata c) external returns (uint256, uint256, byte) {
```

Comment:
Can you add value parameters before and after `c` so that the data pointer gets more complicated?